game-core/src/main/java/games/strategy/engine/framework/startup/ui/panels/main/MainPanel.java

```diff
@@ -90,7 +88,16 @@
         .build();
 
     setLayout(new BorderLayout());
-    addChat();
```

Comment:
Instead of `addChat` knowing whether it should actually add a chat or not, and how to get that chat (violation of both SRP and DI), it is now passed the chat component it should render directly.